sw/device/tests/pmp_sanitytest_napot.c

```diff
@@ -0,0 +1,110 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/runtime/pmp.h"
+
+#include "sw/device/lib/handler.h"
+#include "sw/device/lib/irq.h"
+#include "sw/device/lib/runtime/check.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/test_main.h"
+#include "sw/device/lib/testing/test_status.h"
+
+#define PMP_LOAD_REGION_ID 0
+
+#define PMP_LOAD_RANGE_BUFFER_SIZE 2048
+#define PMP_LOAD_RANGE_SIZE 1024
+#define PMP_LOAD_RANGE_BOTTOM_OFFSET 0
+#define PMP_LOAD_RANGE_TOP_OFFSET 1023
+
+// These flags are used in the test routine to verify that a corresponding
+// interrupt has elapsed, and has been serviced. These are declared as volatile
+// since they are referenced in the ISR routine as well as in the main program
+// flow.
+static volatile bool pmp_load_exception;
+
+/**
+ * The buffer that is used for load/store access violation test.
+ */
+__attribute__((aligned(PMP_LOAD_RANGE_SIZE))) static volatile char
```

Comment:
Consider following the attribute with // to force clang-format to wrap this.

Reply:
Done.